packages/angular_devkit/build_angular/src/utils/webpack-browser-config.ts

```diff
@@ -51,13 +53,10 @@ export async function generateWebpackConfig(
 
   // At the moment, only the browser builder supports differential loading
   // However this config generation is used by multiple builders such as dev-server
-  // const builderInfo = additionalOptions.builderInfo;
-  // const differentialLoading =
-  //   builderInfo
-  //   && builderInfo.builderName === 'browser'
-  //   && isDifferentialLoadingNeeded(projectRoot, scriptTarget);
-  const differentialLoading = false;
-  const scriptTargets = [tsConfig.options.target];
+  const scriptTarget = tsConfig.options.target;
+  const differentialLoading = context.builder.builderName === 'browser'
```

Comment:
Any better idea @clydin?

Reply:
fine for now.  It doesn't affect the public API so we can cleanup and refactor once it's in.